src/compress.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,7 @@
+setup.py
+compress.egg-info/PKG-INFO
+compress.egg-info/SOURCES.txt
+compress.egg-info/dependency_links.txt
+compress.egg-info/top_level.txt
+test/__init__.py
+test/test.py
```

Comment:
Depending on how you would create release packages, it's better to keep most generated out of git. 

✅  `.lock` files are an example where you probably want to check them. They're explicit about the dependencies are and which specific versions to install. Other devs can use it to replicate an environment as closely as possible for testing.
❌  File manifests, IDE settings, any local files (data, configs with sensitive values, etc). Basically anything that quickly becomes outdated or that other developers don't need.